packages/front-end/components/Features/ConditionInput.tsx

```diff
@@ -425,6 +444,18 @@ export default function ConditionInput(props: Props) {
                       containerClassName="col-sm-12 col-md mb-2"
                       required
                     />
+                  ) : ["string", "secureString"].includes(attribute.datatype) &&
+                    attribute.format === "date" &&
+                    !["$regex", "$notRegex"].includes(operator) ? (
+                    <Field
+                      type="datetime-local"
+                      value={format(new Date(value), "yyyy-MM-dd'T'HH:mm")}
```

Comment:
Jeremy recommends using the same field that we're using for regular strings. Determine field type depending on whether the format is date or not. Don't format the value